# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../perl/install.man.patch.cross
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- perl-5.28.0/Makefile.SH.vanilla	2018-11-15 11:55:46.051538740 +0000
+++ perl-5.28.0/Makefile.SH	2018-11-15 11:56:28.047538108 +0000
@@ -1058,14 +1058,14 @@
 
 install_$name install-$name: \$(INSTALL_DEPENDENCE) installperl all installman
 	\$(HOST_PERL) installperl --destdir=\$(DESTDIR) $flags \$(INSTALLFLAGS) \$(STRIPFLAGS)
-	\$(HOST_PERL) installman --destdir=\$(DESTDIR) $flags
+	#\$(HOST_PERL) installman --destdir=\$(DESTDIR) $flags
 EOT
     else
         $spitshell >>$Makefile <<EOT
 
 install_$name install-$name: \$(INSTALL_DEPENDENCE) installperl all installman
 	\$(RUN_PERL) installperl --destdir=\$(DESTDIR) $flags \$(INSTALLFLAGS) \$(STRIPFLAGS)
-	\$(RUN_PERL) installman --destdir=\$(DESTDIR) $flags
+	#\$(RUN_PERL) installman --destdir=\$(DESTDIR) $flags
 EOT
     fi
 
@@ -1219,7 +1219,7 @@
 
 $spitshell >>$Makefile <<!GROK!THIS!
 install.man:	all installman
-	${LOCAL_PERL} installman --destdir=\$(DESTDIR) \$(INSTALLFLAGS)
+	#${LOCAL_PERL} installman --destdir=\$(DESTDIR) \$(INSTALLFLAGS)
 
 # XXX Experimental. Hardwired values, but useful for testing.
 # Eventually Configure could ask for some of these values.
